meson.build

```
@@ -2,7 +2,7 @@ project(
     'granite',
     'vala', 'c',
     meson_version: '>= 0.48.2',
-    version: '5.4.0'
+    version: '5.4.1'
```

probably minor not patch right?
```suggestion
    version: '5.5.0'
```

---

ah yeah derp